### PR TITLE
[DO NOT MERGE] Update create-release-pr.yml

### PR DESCRIPTION
**Pull request by ACoolmanTelicent**

### CODE REVIEW

File: .github/workflows/create-release-pr.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 name: Publish new release version
 
+permissions:
+  contents: write
+  pull-requests: write
+
 on:
   push:
-    branches:
-      - "main"
```
Comment (tatelicent) [inline .github/workflows/create-release-pr.yml:6]: Is this not required any more?

Comment (ACoolmanTelicent) [reply .github/workflows/create-release-pr.yml:6]: This is a quick wip to try and improve the feedback we get from permissions being insufficient

Comment (tatelicent) [reply .github/workflows/create-release-pr.yml:6]: Ah think you might just be doing this for testing

**Comment by ACoolmanTelicent** [comment]
Didn't achieve test, as it seems jobs aren\t run from orgA to orgB, likely due to security

**Pull request closed without merging**